website/docs/d/users.html.markdown

```diff
@@ -31,6 +31,8 @@ The following arguments are supported:
 
 * `mail_nicknames` - (Optional) The email aliases of the Azure AD Users.
 
+* `graceful_errors` - (Optional) Whether Terraform will return an error if any user lookup fails. An error will still occur if no users are returned.
```

Comment:
In addition to the property rename, could we word this:

> (Optional) Ignore missing users and return users that were found. The data source will still fail if no users are found. Defaults to `false`.

Also can we shift this down to L36 (underneath the _NOTE_ about the required search properties)